### PR TITLE
add sample config file

### PR DESCRIPTION
**Pull request by Madhu-1**
add support to run on single ip address

Signed-off-by: Madhu Rajanna <mrajanna@redhat.com>

### CODE REVIEW

File: gluster-block.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+GlusterBlockCLIPath = "/usr/bin/glusterd"
```
Comment (amarts) [inline gluster-block.toml]: This doesn't look right. Shouldn't this be `/usr/bin/gluster-block` ?

Comment (aravindavk) [inline gluster-block.toml]: Not matching `toml` field name

Comment (aravindavk) [reply gluster-block.toml]: Default conf file(`$SRC/extras/conf/config.toml.sample`) already exists and used in `main.go` can that be used?

Comment (Madhu-1) [reply gluster-block.toml]: updated

Comment (Madhu-1) [reply gluster-block.toml]: why config file is in `$SRC/extras/conf/config.toml.sample` it should be in source dir, right?

**Comment by Madhu-1** [comment]
@aravindavk PTAL

File: gluster-block.toml (hunk 1 above)
Comment (aravindavk) [reply gluster-block.toml]: You can move if required.

Comment (aravindavk) [reply gluster-block.toml]: `Makefile` needs change if moved.

Comment (Madhu-1) [reply gluster-block.toml]: kept it in same folder .PTAL

File: extras/conf/config.toml.sample [hunk 2]
```diff
@@ -1,5 +1,5 @@
-gluster-block-cli-path = "gluster-block"
-port = 8081
+gluster-block-cli-path = "/usr/bin/gluster-block"
```
Comment (aravindavk) [inline extras/conf/config.toml.sample]: `/usr/sbin/gluster-block`  https://github.com/gluster/gluster-block/blob/master/gluster-block.spec.in#L74

Comment (Madhu-1) [reply extras/conf/config.toml.sample]: done

**Pull request merged**